frontend/src/components/maps/leaflet/LayerPopup/LayerPopUpContent.test.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { createMemoryHistory } from 'history';
+import { render, cleanup } from '@testing-library/react';
+import { IPopupContentProps, LayerPopupContent } from './LayerPopupContent';
+import { Router } from 'react-router-dom';
+import queryString from 'query-string';
+
+const history = createMemoryHistory();
+jest.mock('hooks/useApi');
+
+const mockLayer: IPopupContentProps = {
+  config: {},
+  data: {
+    feature_area_sqm: '10000',
```

Comment:
Everything being a string highlights the issue I was mentioning in a prior PR.  I mentioned Formik appears to require this, but Devin mentioned it doesn't.  Not asking you to change anything, simply highlighting the ugliness of how it's currently implemented.

Reply:
it won't let me assign it anything but a string

Reply:
Yeah it won't because it has been designed to only allow strings.  You don't need to change anything here.  I'm just highlighting one of the issues we have in our implementation due to either Formik, javascript or something else.